components/ospcommon/cmake/macros.cmake

```diff
@@ -206,6 +223,24 @@ MACRO(OSPRAY_INSTALL_LIBRARY name component)
     ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
       COMPONENT devel
   )
+
+  IF ("${export}" STREQUAL "TRUE")
+    # Export targets to install directory
+    # Note (Qi): this creates one file per target, thus we have to include all of them
+    #   when compiling ospray related applications
```

Comment:
I know we can append target to one single export file if we export to the build tree. But it seems we can not do that if we want to export to the install directory. Is there a reason for that? 

Actually the reason I created multiple export files for each target was because ospray will call install multiple times for each target, I didn't want to change it too dramatically ... But yeah, one single export file will be neat if possible ...